packages/wrangler/src/inspect.ts

```diff
@@ -587,6 +590,7 @@ function logConsoleMessage(evt: Protocol.Runtime.ConsoleAPICalledEvent): void {
   const method = mapConsoleAPIMessageTypeToConsoleMethod[evt.type];
 
   if (method in console) {
+    // @ts-expect-error - we know this is a valid method
```

Comment:
One last thing...
The problem is that `console[method].apply(console, args)` is not actually valid for `console.dir()` and `console.table()`.
So we can avoid this TS error by using:

```ts
  if (method in console) {
    switch (method) {
      case "dir":
        console.dir(args);
        break;
      case "table":
        console.table(args);
        break;
      default:
        // eslint-disable-next-line prefer-spread
        console[method].apply(console, args);
        break;
    }
  }
```